CODEOWNERS

```diff
@@ -1,46 +1,97 @@
+apps/cli/                                                @ledgerhq/live-blockchain-support
+apps/web-tools/                                          @ledgerhq/live-blockchain-support
+docs/                                                    @ledgerhq/live-blockchain-support
+libs/coin-*/                                             @ledgerhq/live-blockchain-support
+libs/domain-service/                                     @ledgerhq/live-blockchain-support
+libs/evm-tools/                                          @ledgerhq/live-blockchain-support
+libs/ledger-live-common/src/families/                    @ledgerhq/live-blockchain-support
+libs/ledgerjs/packages/cryptoassets/                     @ledgerhq/live-blockchain-support
+libs/ledgerjs/packages/hw-app-*/                         @ledgerhq/live-blockchain-support
+libs/ledgerjs/packages/types-cryptoassets/               @ledgerhq/live-blockchain-support
+libs/live-network/                                       @ledgerhq/live-blockchain-support
 
-# PTX
-ledger-live-desktop/src/renderer/screens/exchange/  @ledgerhq/ptx
-ledger-live-mobile/src/screens/Exchange/            @LedgerHQ/ptx
-ledger-live-common/src/exchange/                    @LedgerHQ/ptx
+# PTX team
+apps/ledger-live-desktop/src/renderer/components/ProviderIcon/  @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/components/WebPTXPlayer/  @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/modals/Platform/          @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/modals/StartStake/        @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/reducers/swap.ts          @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/screens/earn/             @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/screens/exchange/         @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/screens/stake/            @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/screens/swapWeb/          @ledgerhq/ptx
+apps/ledger-live-mobile/src/actions/earn.ts                     @ledgerhq/ptx
+apps/ledger-live-mobile/src/actions/swap.ts                     @ledgerhq/ptx
+apps/ledger-live-mobile/src/components/ProviderIcon/            @ledgerhq/ptx
+apps/ledger-live-mobile/src/components/Stake/                   @ledgerhq/ptx
+apps/ledger-live-mobile/src/components/WebPTXPlayer/            @ledgerhq/ptx
+apps/ledger-live-mobile/src/reducers/earn.ts                    @ledgerhq/ptx
+apps/ledger-live-mobile/src/reducers/swap.ts                    @ledgerhq/ptx
+apps/ledger-live-mobile/src/screens/Exchange/                   @ledgerhq/ptx
+apps/ledger-live-mobile/src/screens/PTX/                        @ledgerhq/ptx
+apps/ledger-live-mobile/src/screens/Swap/                       @ledgerhq/ptx
+libs/ledger-live-common/src/exchange/                           @ledgerhq/ptx
 
-# Wallet API
-ledger-live-common/src/platform/                    @LedgerHQ/wallet-api
+# Wallet API team
```

Comment:
[SHOULD] It's connectivity team now no ?

Reply:
mmh at the moment that team still is named like this on github